<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleDialog</class>
 <widget class="QDialog" name="ScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рейс</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="departureCityLabel">
       <property name="text">
        <string>Город отправления:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="departureCityEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="arrivalCityLabel">
       <property name="text">
        <string>Город прибытия:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="arrivalCityEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="departureDateLabel">
       <property name="text">
        <string>Дата отправления:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="departureDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="arrivalDateLabel">
       <property name="text">
        <string>Дата прибытия:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="arrivalDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Стоимость билета (руб):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="priceSpin">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.00000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="seatsLabel">
       <property name="text">
        <string>Доступно мест:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="seatsSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>









